spec/models/form_profile_spec.rb

```diff
@@ -699,9 +699,11 @@
         'addressLine2' => street_check[:street2],
         'city' => user.va_profile[:address][:city],
         'stateCode' => user.va_profile[:address][:state],
-        'countryName' => 'USA',
+        'countryName' => user.va_profile[:address][:country],
         'zipCode' => user.va_profile[:address][:postal_code][0..4]
-      }
+      },
+      'claimantPhoneNumber' => user.va_profile[:address][:state],
```

Comment:
It looks like you have the phone number and email address flip flopped.

Reply:
'claimantPhoneNumber' => user.va_profile[:address][:state] should be 'claimantPhoneNumber' => us_phone and vise versa

Reply:
Oh, and phone number is set to state 'claimantPhoneNumber' => user.va_profile[:address][:state]

Reply:
Geez I am an idiot